DependencyInjection/Compiler/IntegrationPass.php

```diff
@@ -30,11 +30,11 @@ public function process(ContainerBuilder $container)
         }
 
         $def = $container->getDefinition('translation.loader.xliff');
-        if ('%translation.loader.xliff.class%' !== $def->getClass()) {
+	if ('%translation.loader.xliff.class%' !== $def->getClass() && 'Symfony\Component\Translation\Loader\XliffFileLoader' !== $def->getClass()) {
```

Comment:
This was in the original code. It replaces the default XLIFF file loader from the Symfony Translation component with a custom one from the bundle.
The 2 checks are needed as 2.x versions of the component where using a parameter for the loaders class name, while 3.x uses the FQCN directly.